poc/TestOfTestFrameworkByReference/Test.cs

```diff
@@ -263,7 +264,7 @@ private static void ThrowsException()
 
     public class SomethingElse
     {
-        public void NothingReally()
+        public void NothingReally(object value, [CallerArgumentExpression(nameof(value))] string parameter = null)
```

Comment:
Is this required change? The parameters seems not to be used?

Reply:
This method existed previously to show that methods not attributed with a test attribute with get called.

I added the CallerArgumentExpression to confirm/prove that the internal attributes won't clash with the public versions and that everything works like it should.